### PR TITLE
[5.1] Core loader custom element rework

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes
This PR has three parts:

1. The core loader custom element has been rewritten to use an SVG rather than built using CSS.
2. We can now pass `inline="true"` and `size="xx"`, allowing us to have an inline instance, rather than fullscreen.
3. The sample data module now uses the updated core loader, rather than an old animated GIF.

**Why SVG over CSS?**
The original CSS version has loads of hardcoded values, meaning scaling became extremely difficult.

**Why aren't we observing the `inline` or `size` attributes?**
Simply because I do not believe they need observing.

### Testing Instructions
1. Run `npm i`
2. Go to your Joomla backend
4. Install the sample data

### Screenshot
loader2

Thank you to @dgrammatiko for providing the SVG.

### CODE REVIEW

**Comment by Quy** [comment]
Would you consider this to be a fix for #31078 #30099?

**Comment by dgrammatiko** [comment]
Nice one Charlie. Quick question: in the screenshots it seems that the logo is twisted (order of the colours). Was it my fault with the [svg](https://codepen.io/dgrammatiko/pen/rNdNjzr)?

**Comment by C-Lodder** [comment]
@dgrammatiko Good catch, fixed.
@Quy Yes, I suppose it can close those

File: administrator/modules/mod_sampledata/tmpl/default.php [hunk 1]
```diff
@@ -14,20 +14,14 @@
 use Joomla\CMS\Uri\Uri;
```
Comment (Quy) [inline administrator/modules/mod_sampledata/tmpl/default.php]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ef94d646fb8c8304423c0ec36eb3cb54c3f4a017<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42903">issues.joomla.org/tracker/joomla-cms/42903</a>.</sub>

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on ef94d646fb8c8304423c0ec36eb3cb54c3f4a017<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42903">issues.joomla.org/tracker/joomla-cms/42903</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42903">issues.joomla.org/tracker/joomla-cms/42903</a>.</sub>

**Comment by LadySolveig** [comment]
Thank you @C-Lodder and for review and testing @dgrammatiko @fgsw @Quy

**Pull request merged**